blacklight-cornell/app/views/catalog/_show_tools.html.erb

```diff
@@ -18,39 +18,6 @@
 <% if !bookmarks_index %>
 <li><%= render :partial => "bookmark_control" , :locals => {:document => @document} %></li>
 <% end %>
-<!-- TODO: TRANSLATE -->
-<% if citable %>
-  <% citelink_content =  '<i class="fa fa-pencil" aria-hidden="true"></i>' + t('blacklight.tools.cite') %>
-  <li>
-  <%= link_to citation_solr_document_path(:id => @document), {:id => 'citation', :name => 'citationSolr', :class => 'lightboxLink btn btn-default btn-sm', data: {}.merge({ blacklight_modal: "trigger" }) } do %>
-    <%= citelink_content.html_safe %>
-  <% end %>
-  </li>
-<% elsif bookmarks_index || bookbags_index%>
-<%# ************************ %>
-  <li>
-  <img id="loadingSpinner" style="display: none;" src="/img/ajax-loader.gif">
-  </li>
-  <li class="dropdown">
-    <div class="btn-group">
-      <a href="#" class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown" id="cite-menu" aria-haspopup="true" aria-expanded="false"><i class="fa fa-share" aria-hidden="true"></i>Cite <b class="caret"></b></a>
-      <ul class="dropdown-menu" aria-labelledby="cite-menu">
-      <% names = %w[apa chicago cse mla mla8]  %>
-      <%  names.each do |name|  %>
-        <% citation_format = t('blacklight.citation.'+name) %>
```

Comment:
There's a handful of blacklight.citation.* i18n labels in blacklight.en.yml - can any of those be removed now?

Reply:
these are now removed